Matecoin trades (#1)
mate-academy/py-matecoin-trades: README.md

# Matecoin trades

- Read [the guideline](https://github.com/mate-academy/py-task-guideline/blob/main/README.md) before start

> You have made some trades with new cryptocurrency "Matecoin".

Reviewer: ```suggestion
You have made some trades with the new cryptocurrency "Matecoin".
```

In order to calculate profit you loaded json-file `trades.json` with 

> trades description. Content of that file is `JsvsScript Object`

Reviewer 2: Better here list of Javascript Objects

which is very similar to 

> python `dict`, where keys are dates of trades and values are

Reviewer 2: And so - here are without `keys`

if you bought, 
`"sold"` - volume of coins, if you sold and `matecoin_price` -
price for 1 Matecoin in dollars that day.

Example:
```javascript
// trades.json
[
  {
    "bought": "0.00111",
    "sold": null,
    "matecoin_price": "48911.23"
  },
  {
    "bought": null,
    "sold": "0.00058",
    "matecoin_price": "77830.83"
  }
]
```

> Write `calculate_profit` function, that takes name of file with

Reviewer: ```suggestion
Write `calculate_profit` function, which takes the name of the file with
```

trades information, converts inside data into python native data

> structure, counts money profit and current coin account, dumps

Reviewer: ```suggestion
structure, counts money profit in dollars and current coin account, dumps
```
